File: resources/vars.py
import os

def main(dict):
    return {
       "api_host": os.environ['__OW_API_HOST'],
       "api_key": os.environ['__OW_API_KEY'],
       "namespace": os.environ['__OW_NAMESPACE'],
       "action_name": os.environ['__OW_ACTION_NAME'],
       "activation_id": os.environ['__OW_ACTIVATION_ID'],
       "deadline": os.environ['__OW_DEADLINE']
    }

File: resources/vars.php
<?php
function main($args) : array {
    return [
        "api_host" => getenv('__OW_API_HOST'),
        "api_key" => getenv('__OW_API_KEY'),
        "namespace" => getenv('__OW_NAMESPACE'),
        "action_name" => getenv('__OW_ACTION_NAME'),
        "activation_id" => getenv('__OW_ACTIVATION_ID'),
        "deadline" => getenv('__OW_DEADLINE'),
      ];
}
?>
